content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,212 @@
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Insights]
+
+Docs: provide a link to the updated docs
+### Configure precise code navigation for 9 languages in a matter of minutes with auto-indexing
+Code navigation allows you to quickly jump to symbol definitions, find references, dependencies, and more, so you can spend more time shipping code. Precise code navigation is the most powerful version of code navigation and it works cross-repository. We now support 9 languages including Ruby, Rust, Go, Java, Scala, Kotlin, Python, TypeScript, JavaScript. Auto-indexing makes precise code navigation work across all of your code, automatically, and it’s turned on by default for our Cloud customers. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Code Navigation]
+
+Docs: provide a link to the updated docs	
+
+### Access code context while staying within the developer workflow 
+Top used extensions, including code navigation, git-extras, open-in-editor, and search-exports, are now part of the core functionality. These enhancements provide developers code context while staying in the flow within their preferred tools. Additionally, the current Sourcegraph extensions framework will be deprecated so that we can make way for a new model that allows a deeper integration throughout our entire code intelligence platform. Please note, these changes do not impact our IDE extensions. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Integrations]
+
+Docs: provide a link to the updated docs	
+##### 🏗️ High-leverage ways to improve your entire codebase
+
+### Make changes across all of your codebases at enterprise scale with server-side Batch Changes (beta)
+Batch Changes can now run across thousands of repositories with a better development experience. Previously, you had to run src-cli locally to create batch changes. That could take an impractical amount of time or be brittle for large or resource-intensive batch changes. By running Batch Changes server-side you can:
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+[Batch Changes]
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+- New packaging updates
```

Comment:
@abschu Let's chat about what you want to include for the new pricing and packaging